docs/source/devguide.rst

```diff
@@ -21,11 +18,14 @@ Developer's Guide
   Custom libraries
   ****************************************
 
-****************
-Our contributors
-****************
+*****************
+Our collaborators
+*****************
+
+The order of our collaborators is not relevant to the their level of implication:
```

Comment:
Just put them in alphabetical order.

Reply:
Okay - I'll do it on the other repo too.